churner/tests/churner_test.go

```diff
@@ -89,14 +93,54 @@ func teardown() {
 func TestChurner(t *testing.T) {
```

Comment:
This test was not testing anything because no operators have been registered. 
Churner simply exited without any `OperatorsToChurn`